src/workloads/query/LookupUnwind.yml

```diff
@@ -0,0 +1,489 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: |
+  This test exercises the behavior of $lookup when followed by $unwind. We use a single "local"
```

Comment:
Can you please extend this comment a little by adding the bold text:

$lookup when followed by $unwind **that immediately unwinds the lookup result array, which is optimized in the execution engine to avoid materializing this array as an intermediate step.**

else it might not be clear to someone new why this pairing matters (and that only a specific type of $unwind after the lookup matters).